doc/configuration.rst

```diff
@@ -411,6 +411,35 @@ hive resources and 1 mysql resource:
 Note that it was not necessary to specify the 1 for mysql here, but it
 is good practice to do so when you have a fixed set of resources.
 
+.. _retcode-config:
+
+[retcode]
+----------
+
+Configure return codes for the luigi binary. In the case of multiple return
```

Comment:
"Enforce" sounds as if it were a burden. It is a best practice that avoids problems in the future (custom thingies clashing with vendor/Luigi thingies). Various kinds of specifications, when dealing with limited amount of things, do make such RFU provisions.

I meant neither of those ranges to be so small as [0, 4]... Say, [0..127] reserved for Luigi, and [128..255] for anyone who wraps or forks or plugs into it (so they are promised to never conflict with something from vanilla Luigi).